src/main/java/practice/StreamPractice.java

```diff
@@ -12,9 +14,17 @@
      * return min integer value. One more thing - we're interested in even numbers.
      * If there is no needed data throw RuntimeException with message
      * "Can't get min value from list: method_input_list"
+     * @return
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers
+                .stream()
+                .flatMap(string -> Arrays.stream(string.split(",")))
+                .mapToInt(Integer::valueOf)
+                .filter(number -> number % 2 == 0)
+                .min()
+                .orElseThrow(() -> new RuntimeException("Can't get min value from list: "
+                        + "method_input_list"));
```

Comment:
```suggestion
                .orElseThrow(() -> new RuntimeException("Can't get min value from list: "
                        + numbers));
```
this way message will be more informative